theme/horizontalrule.css

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+.ck-content .ck-horizontal-rule {
+	/* Handle proper `<hr>` display next to elements with `float` property, e.g. side image case. */
+	overflow: hidden;
+
+	padding: 5px;
+}
+
+.ck-content .ck-horizontal-rule hr {
```

Comment:
Similar as above – if these are supposed to be content styles then `.ck-horizontal-rule` is just not there. So the only selector that we can use is `.ck-content hr`.